builder/builder-next/exporter/snapshotexporter/snapshotexporter.go

```diff
@@ -0,0 +1,27 @@
+package snapshotexporter
+
+import (
+	"context"
+	"strings"
+
+	mobyexporter "github.com/docker/docker/builder/builder-next/exporter"
+	"github.com/moby/buildkit/exporter"
+)
+
+type snapshotExporter struct {
+	exp exporter.Exporter
+}
+
+func New(exp exporter.Exporter) (exporter.Exporter, error) {
+	return &snapshotExporter{exp: exp}, nil
+}
+
+func (e *snapshotExporter) Resolve(ctx context.Context, ExporterAttrs map[string]string) (exporter.ExporterInstance, error) {
+	reposAndTags, err := mobyexporter.SanitizeRepoAndTags(strings.Split(ExporterAttrs["name"], ","))
+	if err != nil {
+		return nil, err
+	}
+	ExporterAttrs["name"] = strings.Join(reposAndTags, ",")
+	ExporterAttrs["unpack"] = "true"
```

Comment:
If moby wants to have different defaults then I think it is better to update the incoming request rather than rewrite exporters.